app/models.py

```diff
@@ -43,6 +43,22 @@ def insert_roles():
     def __repr__(self):
         return '<Role \'%s\'>' % self.name
 
+class IdlingIncident(db.Model):
+    __tablename__ = "incidents"
+    vehicle_id = db.Column(db.Integer, primary_key=True)
```

Comment:
Ok, so `vehicle_id` is just something government vehicles have. I think that we might just want to ask for license plate instead. @kongsally, can you ask them how they feel about this?

I think license plate is better because

1. Users already know where to find the plate on the car.
2. Agencies can map license plate to vehicle id.
3. We can ask consistent questions in the reporting form (e.g. we do not need to ask if this was a government vehicle)